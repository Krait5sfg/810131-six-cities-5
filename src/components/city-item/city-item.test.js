import React from 'react';
import renderer from 'react-test-renderer';
import CityItem from './city-item';

it(`CityItem render correctly with active feature`, () => {
  const tree = renderer.create(<CityItem
    onCityLinkClick={() => {}}
    city={`Dusseldorf`}
    isActive={false}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});

it(`CityItem render correctly without active feature`, () => {
  const tree = renderer.create(<CityItem
    onCityLinkClick={() => {}}
    city={`Dusseldorf`}
    isActive={true}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
