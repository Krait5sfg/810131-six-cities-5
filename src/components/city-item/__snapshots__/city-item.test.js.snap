// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityItem render correctly with active feature 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item "
    href="#"
    onClick={[Function]}
  >
    <span>
      Dusseldorf
    </span>
  </a>
</li>
`;

exports[`CityItem render correctly without active feature 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="#"
    onClick={[Function]}
  >
    <span>
      Dusseldorf
    </span>
  </a>
</li>
`;
